cast/tests/utils/prepare_for_tests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if ! which starknet-devnet > /dev/null 2>&1; then
+  echo "starknet-devnet not found, exiting."
+  exit 1
+fi
+
+COMPILER_DIRECTORY="$(git rev-parse --show-toplevel)/cast/tests/utils/"
+CAIRO_REPO="https://github.com/starkware-libs/cairo/releases/download"
+COMPILER_VERSION="v1.0.0"
+
+if [ ! -x "$COMPILER_DIRECTORY/starknet-sierra-compile" ]; then
```

Comment:
it does not matter really, I just didn't see the point of having it nested (especially so that it is excluded in gitignore)
However, if for some reason you insist it to be under this path, I have no objections.